main.py

```
@@ -3,6 +3,7 @@
 from dotenv import load_dotenv
 load_dotenv()
 from interview.interface.controllers import interview_controller
+from interview.interface.controllers import stt_controller
```

Pylint (3.3.7)

[convention] Import "from interview.interface.controllers import stt_controller" should be placed at the top of the module

(C0413)